src/documentation/markdown/2017/ts-data-fields.md

```diff
@@ -0,0 +1,72 @@
+## Public Transmittal Sheet data fields with values and definitions
+
+### [activity\_year](#activity_year)
+**Description:** The calendar year of data collection.
+
+**Values:**
+- Integer values for year
+
+### [calendar\_quarter](#calendar_quarter)
+**Description:** The calandar quarter to which the data refer. Quarter 4 is for the annual submission of data.
+
+**Values:**
+- 4
+
+### [lei](#lei)
+**Description:** The Legal Entity Identifier (LEI) of the filing institution.
+
```

Comment:
LEI was not present in 2017.

Respondent ID was present in its place, [see this spec](https://s3.amazonaws.com/cfpb-hmda-public/prod/snapshot-data/2017_publicstatic_dataformat.pdf)